skyportal/handlers/api/tag.py

```diff
@@ -0,0 +1,190 @@
+import re
+
+from sqlalchemy import func
+
+from baselayer.app.access import auth_or_token
+
+from ...models import Obj, ObjTag, ObjTagOption
+from ..base import BaseHandler
+
+
+class ObjTagOptionHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        with self.Session() as session:
+            tags = session.scalars(ObjTagOption.select(session.user_or_token)).all()
+            return self.success(data=tags)
+
+    @auth_or_token
```

Comment:
didn't you mention that only admins could post tags? Looks like there is no rules enforced here (if you look at other handlers, there is a `@permissions([...])` method that takes the allowed permissions to call a handler.